alma/alma-8.x/common/install_pmix.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -ex
+
+PMIX_VERSION=$(jq -r '.pmix."'"$DISTRIBUTION"'".version' <<< $COMPONENT_VERSIONS)
+REPO_DIR="$ALMA_COMMON_DIR/../alma-8.x/common"
+
+cp ${REPO_DIR}/slurmel8.repo /etc/yum.repos.d/slurm.repo
+
+## This package is pre-installed in all hpc images used by cyclecloud, but if customer wants to
+## build an image from generic marketplace images then this package sets up the right gpg keys for PMC.
+if [ ! -e /etc/yum.repos.d/microsoft-prod.repo ];then
+   curl -sSL -O https://packages.microsoft.com/config/rhel/8/packages-microsoft-prod.rpm
+   rpm -i packages-microsoft-prod.rpm
+   rm packages-microsoft-prod.rpm
+fi
+
+dnf config-manager --set-enabled powertools
+yum -y install pmix-$PMIX_VERSION.el8 hwloc-devel libevent-devel munge-devel
```

Comment:
Ok for this PR, but pls add a follow up PR (after release) to move all the dependencies to a single file.